src/recipes/mezcal-with-jalapeno-pepper.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "Mezcal with grapefruit and jalapeno pepper",
+    "description": "To make a toast with friends, we don't need excuses, just a delicious and unique drink to celebrate in any moment.",
+    "github": "ferdox2",
+    "ingredients": [
+        {
+            "quantity": "4",
+            "measure": "Oz",
+            "ingredient": "Mezcal"
+        },
+        {
+            "quantity": "1/2",
+            "measure": "Cup",
+            "ingredient": "Grapefruit Juice"
+        },
+        {
+            "quantity": "1",
+            "measure": "Teaspoon",
+            "ingredient": "Syrup"
+        },
+        {
+            "quantity": "1",
+            "ingredient": "Jalapeno pepper in slices"
+        },
+        {
+            "quantity": "8",
+            "ingredient": "Little coriander branches"
+        },
```

Comment:
Add `"measure": "",`, otherwise this will render as `undefined`. Thanks.

Reply:
done